src/pages/home/release-notes/2022.md

```diff
@@ -158,12 +221,6 @@ With this release, the extension provides APIs that you can use to:
 
 * Fixed an issue where the Target display notification was not being sent to the server, upon invoking `displayedLocations` API, if a prior prefetch call did not return profile state token for the mbox.
 
-## May 24, 2022
```

Comment:
This is already present in acp docs https://github.com/AdobeDocs/acp-mobile-sdkdocs/blob/main/src/pages/documentation/release-notes/2022.md#ios-analytics-252